package.json

```diff
@@ -24,6 +24,7 @@
     "express": "~4.18",
     "express-graceful-exit": "~0.5",
     "fast-myers-diff": "~3",
+    "html-entities": "^2.4.0",
```

Comment:
We already use the `entities` package for decoding. Could we use it for encoding as well?